### ISSUE TITLE
Error while using literallycanvas with React 16

### ISSUE THREAD

**Issue by TRomesh** [literallycanvas/literallycanvas#490]
Im having issues while using literallycanvas with react 16 it keep saying that

**Module not found: Can't resolve '../core/LiterallyCanvas' in '/home/tromesh/App/node_modules/literallycanvas/lib/js/reactGUI'**

This is my code of the component
```
import React, { Component } from "react";
import LC from "literallycanvas";
import LiterallyCanvas from "literallycanvas/lib/js/core/LiterallyCanvas";
import defaultOptions from "literallycanvas/lib/js/core/defaultOptions";

class Whiteboard extends Component {
  constructor(props) {
    super(props);

    defaultOptions.backgroundColor = "#fff";
    defaultOptions.imageURLPrefix = "/lib/img";
    defaultOptions.toolbarPosition = "bottom";
    this.lc = new LiterallyCanvas(defaultOptions);
    this.lc.on("shapeSave", this.SaveShape);
  }
  render() {
    return <LC.LiterallyCanvasReactComponent lc={this.lc} />;
  }
}

export default Whiteboard;
```
It would be great if you provide proper documentations on how to use it with latest version of react.

**Comment by anx-ckreuzberger** [comment]
Hi!
This seems to be related to PR #487 
Please take a look there, upvote the PR and maybe make a comment

**Comment by irskep** [comment]
I just published 0.5.1 to npm, so this should be fixed.
